src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -110,6 +110,7 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         // This event is explicitly revoked in the destructor: does not need weak_ref
         auto onReceiveOutputFn = [this](const hstring str) {
             _terminal->Write(str);
+            _terminal->ClearPatternTree();

```

Comment:
this doesn't make us flicker for every character, right?

Reply:
I knew there was a reason I did not clear the pattern tree here before. I'll see if I can change it to only clear the patterns if the output _overwrites_ existing text

Reply:
Okay its _actually_ fixed now